creds.config

```diff
@@ -0,0 +1,3 @@
+[DEFAULT2]
+username = test
+password = p@ssword
```

Comment:
I was able to still push these changes to the creds.config file when testing. So this solution doesn't work. We're looking for a way to give the user a blank template file but then make sure they cannot accidentally push plain text passwords into git history.

Reply:
if we can pass in sha256 encrypted passwords, that would be ideal

Reply:
how did the gitignore fail? hrmmmm

Reply:
try now

Reply:
I don't think we can just pass in sha256 passwords, as that is not encryption but just a hash. 

I think we should look into getting access tokens. We can use those in place of usernames and passwords and the GitHub library we are using can accept those. I propose we just add a command line argument or read it from the config, but once we do that we can store it in an environment variable and encrypt it using AES or something. That way the user doesn't have to authenticate again, at least for a while. 

[How to create access tokens](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)

Reply:
By the way guys @MisterWhopper is a cyber-security student, so he probably knows quite a bit in this area, haha

Reply:
So if you want @MisterWhopper, you can take the lead on this one?

Reply:
Yeah I will take the lead on this.